test/ansible-inventory/roles/inventory/tasks/main.yml

```diff
@@ -10,4 +10,4 @@
           namespace: '{{ meta.namespace }}'
         data:
           sentinel: '{{ sentinel }}'
-          groups: '{{ groups | to_nice_yaml }}'
+          groups: '{{ groups | to_nice_yaml }}'
```

Comment:
Make sure you're not modifying file whitespace unnecessarily.

Reply:
👍